pkg/tests/tasks/security/authorization/tcp_test.go

```diff
@@ -16,156 +16,195 @@ package authorization
 
 import (
 	"fmt"
-	"strings"
 	"testing"
-	"time"
 
-	"github.com/maistra/maistra-test-tool/pkg/examples"
-	"github.com/maistra/maistra-test-tool/pkg/util"
-	"github.com/maistra/maistra-test-tool/pkg/util/log"
+	"github.com/maistra/maistra-test-tool/pkg/app"
+	"github.com/maistra/maistra-test-tool/pkg/util/check/assert"
+	"github.com/maistra/maistra-test-tool/pkg/util/hack"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/pod"
+	"github.com/maistra/maistra-test-tool/pkg/util/retry"
 	"github.com/maistra/maistra-test-tool/pkg/util/test"
 )
 
-func cleanupAuthorTCP() {
-	log.Log.Info("Cleanup")
-	util.KubeDeleteContents("foo", TCPDenyGETPolicy)
-	util.KubeDeleteContents("foo", TCPAllowGETPolicy)
-	util.KubeDeleteContents("foo", TCPAllowPolicy)
-	echo := examples.Echo{Namespace: "foo"}
-	echo.UninstallWithProxy()
-	sleep := examples.Sleep{Namespace: "foo"}
-	sleep.Uninstall()
-	time.Sleep(time.Duration(20) * time.Second)
+// TestAuthorizationTCPTraffic validates authorization polices for TCP traffic.
+func TestAuthorizationTCPTraffic(t *testing.T) {
+	test.NewTest(t).Id("T21").Groups(test.Full, test.InterOp).Run(func(t test.TestHelper) {
+		hack.DisableLogrusForThisTest(t)
+
+		ns := "foo"
+		t.Cleanup(func() {
+			oc.RecreateNamespace(t, ns)
```

Comment:
We need to recreate mesh namespace too, because is going to be set in the setup of each execution. With this we avoid left configurations between test

Reply:
well, this test case all configurations are applied in the foo namespace. I am not aware of any mesh namespace change yet. 
Ok, I can add a "oc.RecreateNamespace(t, meshNamespace)" in this PR

Reply:
Yes, but we are deleting in every test the smcp namespace because is going to create again. I know in this case seems to not apply